service/service_test.go

```diff
@@ -332,14 +329,14 @@ func TestClose(t *testing.T) {
 
 			svcErrors := make(chan error, 1)
 			svcList := service.NewServiceList(initMock)
-			svc, err := service.Run(ctx, cfg, svcList, testBuildTime, testGitCommit, testVersion, svcErrors, testIdentityClient, taskNames)
+			svc, err := service.Run(ctx, cfg, svcList, testBuildTime, testGitCommit, testVersion, svcErrors, testIdentityClient, taskNames, testSearchClient)
 			So(err, ShouldBeNil)
 
 			err = svc.Close(context.Background())
 			So(err, ShouldNotBeNil)
 			So(hcMock.StopCalls(), ShouldHaveLength, 1)
 			So(failingServerMock.ShutdownCalls(), ShouldHaveLength, 1)
-			So(mongoDbMock.CloseCalls(), ShouldHaveLength, 1)
+			So(mongoDBMock.CloseCalls(), ShouldHaveLength, 1)
 		})
 	})
 }
```

Comment:
don't we need some tests for the new search service client health check?

Reply:
I think these are now covered?

Reply:
more or less by producer_test I think